src/pages/index.html

```diff
@@ -0,0 +1,504 @@
+            width="100"
+            height="95"
+            viewBox="0 0 100 95"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <path
+                d="M0.000131319 0.218966L99.9999 -0.000102199C100.114 51.9179 55.4379 94.0978 0.206055 94.2188L0.000131319 0.218966Z"
+                fill="#008EFF"
+            />
+        </svg>
+    </div>
+    <div class="arrow-half">
+        <svg
+            width="48"
+            height="63"
+            viewBox="0 0 48 63"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <path
+                d="M5.50764e-06 -3.8147e-06L0 63L48 31.4956L5.50764e-06 -3.8147e-06Z"
+                fill="#12B74A"
+            />
+        </svg>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 offset-lg-1 col-lg-5">
+                <div class="stract-about position-relative">
+                    <img
+                        class="w-100"
+                        src="assets/images/about-starct.jpg"
+                        alt=""
+                    />
+                    <img
+                        class="position-absolute stract-img"
+                        src="assets/images/about-starct-1.jpg"
+                        alt=""
+                    />
+                    <div class="circle-player">
+                        <img class="player-img" src="assets/images/player.svg" alt="" />
+
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-lg-6">
+                <div class="stract-about-two">
+                    <h2>
+                        Know More About <br />
+                        Stract
+                    </h2>
+                    <p>
+                        Stract is very important because of some security
+                        threats and cyber-attacks. This software protects the
+                        data. Cybersecurity is important because not only it
+                        helps to secure information but also our system from
+                        virus attack. Cybersecurity is important because not
+                        only it helps to secure information but also our system.
+                    </p>
+                    <div class="rating d-flex align-items-center">
+                        <div class="rating-all d-flex flex-column">
+                            <span class="number-rating">4.5/5</span>
+                            <span class="text-rating">Customer Rating</span>
+                        </div>
+                        <div class="rating-all d-flex flex-column">
+                            <span class="number-rating">425</span>
+                            <span class="text-rating">Security Testing</span>
+                        </div>
+                    </div>
+                    <button type="submit" class="btn-stract">Learn More</button>
```

Comment:
@shaban011147 @M-Khairy-B Again what is this?

Reply:
Done